docs/build/guides/transactions/pooled-accounts-muxed-accounts-memos.mdx

```diff
@@ -58,204 +58,313 @@ Not all operations can be used with muxed accounts. Here is what you can use the
+
+### Example 2 - Muxed to Unmuxed Payment (M → G)
+
+This demonstrates a payment from a muxed account (representing a custodial customer) to a regular Stellar account. The underlying custodian account executes the payment, but the transaction includes the muxed address to identify which customer initiated it.
+
+The transaction is signed with custodian keys since muxed accounts have no secret keys. The custodian account balance decreases by 10 XLM plus fees.
+
+### Example 3 - Muxed to Muxed Payment (M → M)
+
+When two muxed accounts share the same underlying account, payments between them are essentially the account sending to itself. The payment amount (+10 XLM and -10 XLM) cancels out, but transaction fees are still charged since the operation is recorded on the ledger.
+
+The account balance decreases only by the transaction fee. You may want to detect these transactions in your application to avoid unnecessary fees. Note that payments between muxed accounts with different underlying accounts would behave like normal G-to-G payments.
+
+---
+
+## Code Implementation
+
+_The following code demonstrates all three examples using Stellar RPC. You can run all examples sequentially with `runAllMuxedExamples()` or execute individual examples using their respective functions._
 
 <CodeExample>
 
 ```js
-const sdk = require("stellar-sdk");
+import * as sdk from "@stellar/stellar-sdk";
 
-const passphrase = "Test SDF Network ; September 2015";
-const url = "https://horizon-testnet.stellar.org";
-let server = new sdk.Server(url);
+const server = new sdk.rpc.Server("https://soroban-testnet.stellar.org");
 
-const custodian = sdk.Keypair.fromSecret(
-  "SAQLZCQA6AYUXK6JSKVPJ2MZ5K5IIABJOEQIG4RVBHX4PG2KMRKWXCHJ",
-);
-const outsider = sdk.Keypair.fromSecret(
-  "SAAY2H7SANIS3JLFBFPLJRTYNLUYH4UTROIKRVFI4FEYV4LDW5Y7HDZ4",
-);
+const custodian = sdk.Keypair.random();
+const outsider = sdk.Keypair.random();
+
+let custodianAcc, outsiderAcc, customers;
 
+// Helper function to format account ID with label
+function formatAccount(accountId) {
+  const shortId = accountId.substring(0, 8);
+  if (accountId === custodian.publicKey()) {
+    return `${shortId} (Custodian)`;
+  } else if (accountId === outsider.publicKey()) {
+    return `${shortId} (Outsider)`;
+  }
+
+  // Check if it's a muxed account by finding the matching customer
+  const matchingCustomer = customers?.find(
+    (customer) => customer.accountId() === accountId,
+  );
+  if (matchingCustomer) {
+    return `${accountId.substring(0, 8)}...${accountId.slice(
+      -6,
+    )} (Customer ${matchingCustomer.id()})`;
+  }
+  return shortId;
+}
+
+function scaleAsset(x) {
+  return Number(x) / 10000000; // Preserves decimal precision
```

Comment:
This changed, no? You need to divide first, otherwise the `Number` cast truncates if larger than $2^{52}-1$ ([docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER)).

Reply:
Pretty sure it's not relevant for holdings < a trillion of a unit so not a big deal but maybe my napkin math is off